pkg/vault/test/authorizer/TimelockAuthorizer.test.ts

```diff
@@ -731,6 +734,192 @@ describe('TimelockAuthorizer', () => {
     });
   });
 
+  describe('addCanceler', () => {
+    context('when the sender is the root', () => {
+      let scheduledId = bn(0);
+      beforeEach('set sender', async () => {
+        from = root;
```

Comment:
This is very much a personal preference thing, but in this case I'm not a fan of this style (which is the style the rest of the test suite follows). We don't have parametrized tests that receive a `from` argument, we simply call from `root` (which is the only valid sender). Given that, I'd instead not have this block, and simply do `{ from: root }` in each call in each test.